tests/packagedcode/data/rpm/header/FaxMail-2.3-12mdv2007.0.src.rpm-expected.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Again this duplicate of https://github.com/nexB/scancode-toolkit/pull/1534/commits/5c3cf55a624dc6775b39dccb7c9760ae04a5bef2 .